examples/c/Makefile.individual

```diff
@@ -0,0 +1,10 @@
+BINARY := $(notdir $(realpath .))
+TARGET := $(BINARY)/$(BINARY)
+
+.PHONY: all
+all:
+	@make --no-print-directory -C .. $(TARGET)
```

Comment:
You should use `$(MAKE)` instead, there is a special section(s) covering this in Make manual :)